vro-types/vrotsc-annotations/index.d.ts

```diff
@@ -244,6 +244,19 @@ interface VroWorkflowItemMethodDecorator {
 	(target: Object, propertyKey?: string | symbol, descriptor?: TypedPropertyDescriptor<Function>): void;
 }
 
+// ---------------------------------------------- Workflow Default Error Handler Canvas Item ------------------------------------------------
+
+export declare const DefaultErrorHandler: VroWorkflowDefaultErrorHandlerDecorator;
+
+interface VroWorkflowDefaultErrorHandlerDecorator {
+	(obj?: VroItemConfiguration): VroWorkflowDefaultErrorHandlerMethodDecorator;
+	new(obj?: VroItemConfiguration): VroItemConfiguration;
```

Comment:
Each item should have their own definition so they are easy to change and don't bring confusion

Reply:
It does not make sense to have own definition in this case, as it does not bring new properties.

Reply:
So each other type have their own Config Type, but specifically the `DefaultErrorhanlder` will point to the `Item` one? Isn't it more clear for the user if it has it's own?

Reply:
Discussed offline, decided that the whole interface will be reworked using inheritance once all needed workflow components are implemented (will be done in a separate PR).